ui/src/app/pages/workspace/workspace-edit-text.tsx

```diff
@@ -160,72 +160,72 @@ export const researchPurposeQuestions: Array<ResearchPurposeQuestion> = [
   }, {
     header: 'What are the anticipated findings from the study? How would your findings ' +
     'contribute to the body of scientific knowledge in the field?',
-    description: <div>Free text; 1000 Character limit
+    description: <div>Free text; 1000 character limit
     </div>
   }, {
-    header: <div>The <i>All of Us </i> Research Program encourages researchers to disseminate their
+    header: <div>The <AoU/> Research Program encourages researchers to disseminate their
       research findings to both scientific and lay/community audiences, to maximize the value of the
       resource and to recognize the contributions of participant partners. Please tell us how you
       plan to disseminate your research findings. Choose as many options below as applicable, and
       specify details in the text box, if available. </div>,
     description: ''
   }, {
-    header: <div>The <i>All of Us</i> Research Program would like to understand how (or if)  your
-      research outcome may fit into the <i>All of Us</i> Research Program Scientific Framework.
+    header: <div>The <AoU/> Research Program would like to understand how (or if)  your
+      research outcome may fit into the <AoU/> Research Program Scientific Framework.
       Please select all of the statements below that describe the outcomes you anticipate from your
       research.</div>,
     description: ''
   }, {
     header: 'Population of interest',
-    description: <div>A primary aim of <i>All of Us</i> is to engage communities that have been
+    description: <div>A primary aim of <AoU/> is to engage communities that have been
       historically underrepresented in biomedical research. The next set of questions is designed
       to help us assess how well we are serving underrepresented communities—and to protect against
-      potentially stigmatizing research findings, to which these groups are more vulnerable.
-
-      <i>All of Us</i> supports well-designed and responsible research that addresses questions relevant to
+      potentially stigmatizing research findings, to which these groups are more
+      vulnerable. <AoU/> supports well-designed and responsible research that addresses questions relevant to
       underrepresented communities, as well as research that compares different groups and
       populations. At the same time, we recognize that research with underrepresented populations
-      can unintentionally result in harms, including the exacerbation of stigma. We encourage
+      can unintentionally result in harm, including the exacerbation of stigma. We encourage
       researchers to consider whether their research design, methods, and analyses could
       potentially cause harm to individuals, groups, and communities.</div>
   }, {
     header: 'Request for Review of Research Purpose Description',
-    description: <div>Any research that focuses on certain population characteristics or
+    description: <span>
```

Comment:
This is NOT being used anywhere, i will remove it in another PR. The same text is in workspace-edit.tsx